WordOfTheDay.Domain/WordsServices.cs

```diff
@@ -7,7 +8,7 @@
 
 namespace WordOfTheDay.Domain
 {
-    internal sealed class WordsServices : IWordsServices
+    public class WordsServices : IWordsServices
```

Comment:
If you made this public anly for testing purposes, consider this solution https://stackoverflow.com/a/58471133/4121489 instead